modules/core/shared/src/main/scala/io/circe/Json.scala

```diff
@@ -228,6 +257,14 @@ sealed abstract class Json extends Product with Serializable {
 
 object Json {
 
+  /**
+   * Merge strategy used for deepMerge operation when merging two arrays.
+   */
+  object MergeMode extends Enumeration {
```

Comment:
I think it'd be better to use the `sealed trait` + `case object` pattern here, for consistency.

Reply:
> I think it'd be better to use the `sealed trait` + `case object` pattern here, for consistency.

done.